src/components/ReservationListCard/ReservationListCard.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled, { css } from 'styled-components/macro';
+
+const ReservationListCard = ({
+  id,
+  name,
+  price,
+  src,
+  origin,
+  destination,
+  departure_time,
+  arrival_time,
+  duration,
+  flight_name,
+}) => {
+  const navigate = useNavigate();
+  const goToPayment = () => {
+    navigate(`/payment/${id}`);
+  };
+
+  const [time, minute] = duration.split(':');
+
+  const convertTime = time => {
+    return (time =
+      time[0] === '0' ? `${time[1]}` + '시간' : `${time}` + '시간');
+  };
+
+  const convertMinute = minute => {
+    if (minute === '00') {
+      return '';
+    } else if (minute[0] === '0') {
+      return ` ${minute[1]}` + '분';
+    } else {
+      return ` ${minute}` + '분';
```

Comment:
num으로 바꾼뒤 0은falsy값이기 때문에 아래와 같이 표현해 볼 수 있을 것 같아요.
여기 뿐만 아니라 다른 곳에서도 사용한다면, utils.js 로 분리해서 이용하는 것도 좋은 방법일 것 같아요!
그리고, 이런 포멧팅 부분을 지원해주는 라이브러리도 많이 사용되고 있답니다.
```suggestion
const convertMinute = minute => {
    if (!Number(minute)) {
      return '';
    } else {
      return `${Number(minute)}분`;
    }
```